filter/classes/classes.go

```diff
@@ -63,7 +63,7 @@ func hasClassMatching(needle string, stack []string) bool {
 
 func hasClass(needle string, stack []string) bool {
 	for _, class := range stack {
-		if class == needle {
+		if class == strings.ToLower(needle) {
```

Comment:
maybe do `lneedle := strings.ToLower(needle)` above the for loop then use `if class == lneedle`

Reply:
Done!